recipes/xdoctest/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "xdoctest" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 71951c60bb8b15fdf3c368c5636a6bbbd658ef5c2e5949e784adf5e1a1275fbd
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - xdoctest = xdoctest.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
```

Comment:
Is `setuptools` needed here? Can you please try without it?

Reply:
I've added it because `conda skeleton pypi` had suggested it, but CI also passes without it. I've removed it again.